streamwebs_frontend/streamwebs/forms.py

```diff
@@ -260,3 +260,18 @@ class StatisticsForm(forms.Form):
         widget=forms.DateInput(attrs={'class': 'datepicker'}),
         label=_('ending on'), required=False
     )
+
+
+class AdminPromotionForm(forms.Form):
```

Comment:
The promo form elements look good and work well, but I worry how they will work when you have thousands of users (as we will in production). A method of searching for a particular user would be very useful.